src/FuzzBase.sol

```diff
@@ -6,7 +6,9 @@ import "./Constants.sol";
 import "./AssertWrapper.sol";
 import "./ClampWrapper.sol";
 import "./IHevm.sol";
+import "./IStdCheats.sol";
 
 abstract contract FuzzBase is AssertWrapper, ClampWrapper, Logging, Constants {
-    IHevm internal vm = IHevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+    IHevm internal vm = IHevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D); // for echidna
+    IStdCheats internal cheats = IStdCheats(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D); // for medusa
```

Comment:
i would like to have some unified interface to `vm` in the longterm, but for now it makes sense to have two

what about renaming `cheats` to `mvm`? (medusa vm)

we could also rename `vm` but i prefer to keep it the same for compatibility reasons

Reply:
I agree with this🙂

Reply:
I agree too, I will rename `cheats` to `mvm`.